src/frontend/src/catalog/table_catalog.rs

```diff
@@ -172,6 +176,9 @@ impl TableCatalog {
             owner: self.owner,
             properties: self.properties.clone(),
             fragment_id: self.fragment_id,
+            vnode_col_idx: self
+                .vnode_col_idx
+                .map(|i| ProstColumnIndex { index: i as _ }),
```

Comment:
Should we also use `ColumnIndex` for keys? 😄

Reply:
It is just for express a option in proto